app/models/manageiq/providers/cloud_manager/template.rb

```diff
@@ -72,7 +72,8 @@ def self.import_image_queue(userid, ext_management_system, options = {})
       :role        => 'ems_operations',
       :zone        => ext_management_system.my_zone,
       :queue_name  => ext_management_system.queue_name_for_ems_operations,
-      :args        => [ext_management_system.id, options]
+      :args        => [ext_management_system.id, options],
+      :msg_timeout => options["timeout"]&.to_i&.hours&.seconds
```

Comment:
@iv1111 does the `ImageImportWorkflow` run directly in the `raw_import_image` queue item?

I thought it was run separately by the ImportDispatcher which queued a `:start` (here https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_runner_workflow.rb#L25-L27)

Reply:
Yes, that is the way it works but the API handler calls this base method that I've modified ... I guess I confused a bunch of stuff.

Reply:
I think it is this last queue_signal that you need to set the msg_timeout on

Reply:
Since the first entry point is this method here, don't I have to propagate the `options["timeout"]` from here somehow further down the chain?